### PR TITLE
Create Rajani

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @gusshawstewart
- [ ] @arunarackal83
- [ ] @ericalloyd
- [ ] @Derek-Chang-BCBSA
- [ ] @RajaniNishanth
- [ ] @rajeshrajanala
- [ ] @Naseer2311
- [ ] @cs07886
- [ ] @Anudeepcse
- [ ] @srinivasgajarla
- [ ] @ramvee321

### PR DESCRIPTION
**Pull request by RajaniNishanth**
TL;DR
fixes #2

Completed:
-commits
-created a branch

To do
 Merge
 Example
cc: @githubteacher

### CODE REVIEW

**Comment by RajaniNishanth** [comment]
TL;DR...

- [ ]

**Review by RajaniNishanth — COMMENTED** [review]
This is awesome!

File: Rajani [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello
```
Comment (RajaniNishanth) [inline Rajani:1]: Change to Hi

Comment (arunarackal83) [inline Rajani:1]: Nice work 👍

Comment (arunarackal83) [inline Rajani:1]:
```suggestion
Hello!
```

**Review by arunarackal83 — APPROVED** [review]

**Pull request merged**